book/src/external-snapshotter.md

```diff
@@ -10,7 +10,8 @@
 
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | [Min K8s Version](kubernetes-compatibility.md#minimum-version) | [Max K8s Version](kubernetes-compatibility.md#maximum-version) | [Recommended K8s Version](kubernetes-compatibility.md#recommended-version) |
 --|--|--|--|--|--|--|--
-[external-snapshotter v4.2.1](https://github.com/kubernetes-csi/external-snapshotter/releases/tag/v4.2.1) | [release-4.2](https://github.com/kubernetes-csi/external-snapshotter/tree/release-4.2) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | k8s.gcr.io/sig-storage/csi-snapshotter:v4.2.1 | v1.20 | - | v1.22
+[external-snapshotter v5.0.1](https://github.com/kubernetes-csi/external-snapshotter/releases/tag/v5.0.1) | [release-5.1](https://github.com/kubernetes-csi/external-snapshotter/tree/release-5.1) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.5.0) | - | k8s.gcr.io/sig-storage/csi-snapshotter:v5.0.1 | v1.21 | - | v1.23
```

Comment:
The min version should be 1.20. @xing-yang do you know why recommended on the tag is 1.20 but for v4.2.1 recommended is 1.22?

Reply:
Hmm, I'm not sure.  In 1.22, there are those API deprecations.  So we should probably recommend 1.22.

Reply:
which API is deprecated?

Reply:
Actually it is deprecated API removal in 1.22, i.e., ValidatingWebhookConfiguration.

Reply:
Both v5 and v4 versions of the snapshotter use `v1.ValidatingWebhookConfiguration` so they are not impacted by the removal.

Reply:
It is not, but this deprecated API removal in 1.22 is a big change that breaks a lot of applications.  At least our recommended version should be something like 1.20+ rather than just 1.20.

Reply:
I think the "+" is implied, but we can add a footnote about this in our page that defines what "recommended" means. It's a bit more nuanced than just "+". For example, if you have v1.0 that recommends 1.15 and then v1.1 that recommends 1.17, then really the recommend range for v1.0 becomes "1.15-1.16".